api/index.js

```diff
@@ -122,7 +123,7 @@ async function doSearch(term) {
         throw e;
     }
 
-    return parseMemeBody(body);
+    return parseMemeBody(body, resultUrl);
```

Comment:
For future proofing's sake, this function may be better with a single object parameter, and just destructure the variables in the params